From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: John Kleinschmidt <jkleinsc@github.com>
Date: Wed, 6 Nov 2019 14:35:31 -0500
Subject: expose_mksnapshot

Needed in order to target mksnapshot for mksnapshot zip.

diff --git a/BUILD.gn b/BUILD.gn
index 3bd3c34aa36187de922b6073749c01d6d4209355..098a60fc75f1472034c35a761b8d226b676042e8 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -3904,7 +3904,6 @@ if (current_toolchain == v8_generator_toolchain) {
 
 if (current_toolchain == v8_snapshot_toolchain) {
   v8_executable("mksnapshot") {
-    visibility = [ ":*" ]  # Only targets in this file can depend on this.
 
     sources = [
       "src/snapshot/embedded/embedded-file-writer.cc",
